formatTest/typeCheckedTests/expected_output/knownMlIssues.re

```diff
@@ -12,12 +11,11 @@ let blahCurriedX x =>
   fun
   | Red 10
   | Black 20
-  | Green 10 => 1
-  /* After or pattern green */
+  | Green 10 =>
+    1 /* After or pattern green */
```

Comment:
Shouldn't this remain as an end of line comment on the entire case? This is not formatted as an end of line comment on the `1`, which changes the meaning. (The code before did not preserve correctly either, but I was under the impression that your latest approach preserves end of line comments, and their meaning).